content/en/cosign/openid_signing.md

```diff
@@ -99,34 +68,22 @@ In order to impersonate an IAM service account, your account must have the `role
 3. Set the `GOOGLE_SERVICE_ACCOUNT_NAME` environment variable to the name of the target account in your cloudbuild.yaml
 4. Sign images in GCB, without keys!
 
-### Timestamps
-
-Signature timestamps are checked in the [rekor](https://github.com/sigstore/rekor) transparency log. Rekor's `IntegratedTime` is signed as part of its `signedEntryTimestamp`. Cosign verifies the signature over the timestamp and checks that the signature was created while the certificate was valid.
-
-## Upcoming work
-
-* Root CA hardening: We should use intermediate certs rather than the root, and support chained verification.
-* Root CA configuration: We should allow users to change the roots and add their own.
-* Other timestamps: We should allow for other timestamp attestations, including attached [RFC3161](https://www.ietf.org/rfc/rfc3161.txt) signatures.
-* Probably a lot more: This is very experimental.
-* More OIDC providers: Obvious.
-
 ## Custom infrastructure
 
 If you're running your own sigtore services flags are available to set your own endpoint's, e.g
 
 ```
- COSIGN_EXPERIMENTAL=1 go run cmd/cosign/main.go sign -oidc-issuer "https://oauth2.example.com/auth" \
-                        -fulcio-url "https://fulcio.example.com" \
-                        -rekor-url "https://rekor.example.com"  \
+ Cosign sign -oidc-issuer "https://oauth2.example.com/auth" \
+                        --fulcio-url "https://fulcio.example.com" \
+                        --rekor-url "https://rekor.example.com"  \
                         ghcr.io/jdoe/somerepo/testcosign
 
 ```
 
-### Custom root cert
+### Custom roots of trust
```

Comment:
Just realized this is already documented in https://docs.sigstore.dev/cosign/custom_components/. Can we link to this in "Custom roots of trust" and remove the duplicated environment variable docs?